configure.ac

```diff
@@ -203,6 +204,20 @@ AC_CHECK_HEADERS([lzma.h],[
 AC_SUBST(WITH_LZMA_LIB)
 
 #=================
+# Check for zstd library.
+
+dnl # Facebook Zstd
+RPM_CHECK_LIB(
```

Comment:
`RPM_CHECK_LIB` makes it work when pkgconfig isn't available (macOS, etc.). It also makes it easy to support internal or external libraries for rpm, which can be handy for stuff where relying on shared libraries might get dangerous.